pkg/geo/geomfn/rotate.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geomfn
+
+import (
+	"math"
+
+	"github.com/cockroachdb/cockroach/pkg/geo"
+	"github.com/twpayne/go-geom"
+)
+
+// Rotate returns a modified Geometry whose coordinates are rotated around the origin by a rotation angle 
+func Rotate(geometry geo.Geometry, rotRadians float64) (geo.Geometry, error) {
+	g, err := geometry.AsGeomT()
+	if err != nil {
+		return geo.Geometry{}, err
+	}
+
+	g, err = RotateGeom(g, rotRadians)
+	if err != nil {
+		return geo.Geometry{}, err
+	}
+
+	return geo.MakeGeometryFromGeomT(g)
+}
+
+// RotateGeom returns a modified geom.T whose coordinates are rotated around the origin by a rotation angle 
+func RotateGeom(g geom.T, rotRadians float64) (geom.T, error){
```

Comment:
any reason this is public?
can you rename this to `rotateGeomT`?

Reply:
fixed in recent commit.